Mesh_3/include/CGAL/make_mesh_3.h

```diff
@@ -94,36 +92,29 @@ namespace parameters {
   // -----------------------------------
   // Features_options
   // -----------------------------------
-  inline internal::Features_options
-  features() { return internal::Features_options(true); }
+  inline Named_function_parameters<internal::Features_options, internal_np::features_option_param_t>
```

Comment:
This for example is what I'm calling a "named parameter" specific code.

Reply:
Sorry about that, will make necessary changes